tests/mr/device/mr_ref_test.hpp

```diff
@@ -166,8 +166,8 @@ inline void test_various_allocations(resource_ref ref)
   }
 }
 
-inline void test_various_async_allocations(rmm::device_async_resource_ref ref,
-                                           cuda_stream_view stream)
+inline void test_various_runtime_async_allocations(rmm::device_async_resource_ref ref,
```

Comment:
I don't see why the function names in this header need to change. These functions are used to test all device_async resources, not just cudaMallocAsync.  So it's not clear to me what adding the word "runtime" means for these functions.

Reply:
I don't know why I renamed these, reverting